public/index.html

```diff
@@ -10,6 +10,12 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
```

Comment:
This is not recommended way of using any third party library!
Check installation instructions for using the css for `semantic-ui-react`